CMakeLists.txt

```diff
@@ -5,122 +5,202 @@ set(CMAKE_CXX_STANDARD_REQUIRED on)
 
 project(EasyRPG_Player CXX C)
 
-# liblcf
-# - ${LIBLCF_PATH}
-
-set(DEFAULT_LIBLCF_PATH "${CMAKE_CURRENT_SOURCE_DIR}/lib/liblcf")
-if(NOT LIBLCF_PATH)
-  set(LIBLCF_PATH "${DEFAULT_LIBLCF_PATH}")
-endif()
-find_package(Git REQUIRED)
-if(NOT EXISTS ${LIBLCF_PATH})
-  execute_process(COMMAND ${GIT_EXECUTABLE} clone "--depth=1"
-    "https://github.com/EasyRPG/liblcf.git"
-    "${LIBLCF_PATH}")
-endif()
-add_subdirectory(
-  ${LIBLCF_PATH}/builds/cmake
-  ${CMAKE_CURRENT_BINARY_DIR}/CMakeFiles/liblcf)
-link_directories(${ICU_LIBRARY_DIRS})
-
 set(EXECUTABLE_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR}/bin)
 set(LIBRARY_OUTPUT_PATH ${CMAKE_CURRENT_BINARY_DIR}/lib)
-#link_directories(${LIBRARY_OUTPUT_PATH})
-#cmake_policy(CMP0015 NEW)
 
 list(APPEND CMAKE_MODULE_PATH
-	"${CMAKE_CURRENT_SOURCE_DIR}/lib/liblcf/builds/cmake/Modules"
 	"${CMAKE_CURRENT_SOURCE_DIR}/builds/cmake/Modules")
 
```

Comment:
Some boilerplate I would like to propose to be added here:
```cmake
# Set a default build type if none was specified
if(NOT CMAKE_BUILD_TYPE)
	message(STATUS "Setting build type to 'Debug' as none was specified.")
	set(CMAKE_BUILD_TYPE Debug CACHE STRING "Choose the type of build." FORCE)
	set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
endif()
```
This makes sure a build type is defined, so different flags can be used.
Cmake will for example use `-g` for "Debug" builds and optimize `-O3` for "Release" build by default.